CHANGES.md

```diff
@@ -41,6 +41,7 @@ Release Notes.
 * Fix: Envoy error logs are not persisted when no metrics are generated
 * Fix: Memory leakage of low version etcd client. [fix-issue](https://github.com/jurmous/etcd4j/pull/185)
 * Allow multiple definitions as fallback in metadata-service-mapping.yaml file.
+* Fix: NPE when configmap has no data. [fix-issue](https://github.com/apache/skywalking/pull/6948)
```

Comment:
```suggestion
* Fix: NPE when configmap has no data. 
```

SkyWalking native issue should not be linked in the changelog. Our git commit log already provides this.

Reply:
ok, Thx.